auth-api/src/auth_api/services/affiliation.py

```diff
@@ -113,9 +106,21 @@ def find_affiliated_entities_by_org_id(org_id):
             else:
                 filtered_affiliations.append(entity)
 
-        current_app.logger.debug('>find_affiliations_by_org_id')
+        current_app.logger.debug('>find_visible_affiliations_by_org_id')
         return filtered_affiliations
 
+    @staticmethod
+    def find_affiliations_by_org_id(org_id):
+        """Return business affiliations for the org."""
+        data = []
+        affiliation_models = AffiliationModel.find_affiliations_by_org_id(org_id)
+        if affiliation_models is None:
+            raise BusinessException(Error.DATA_NOT_FOUND, None)
```

Comment:
is it intentional to raise exception? instead of empty list?

Reply:
Good catch, will remove it

Reply:
Fixed in new PR